ios/NotificationService/NotificationService.swift

```diff
@@ -94,11 +95,18 @@ class NotificationService: UNNotificationServiceExtension {
       if isCRTEnabled && !rootId.isEmpty {
         conversationId = rootId
       }
+      
+      if channelName == nil && message == "",
+         let senderName = senderName,
+         let body = bestAttemptContent?.body {
+        message = body.replacingOccurrences(of: "\(senderName) ", with: "")
```

Comment:
I don't completely understand this particular step.

Reply:
Basically the body of the notification has the following format `Sender name mentioned you.` the idea is to remove the sender name from the body of the notification so that it reads `mentioned you.` while the title of the notification is set to the sender name.